llama-index-integrations/llms/llama-index-llms-sambanova/llama_index/llms/sambanova/__init__.py

```diff
@@ -1,3 +1,3 @@
-from llama_index.llms.sambanova.base import Sambaverse, SambaStudio
+from llama_index.llms.sambanova.base import SambaNovaCloud
 
-__all__ = ["Sambaverse", "SambaStudio"]
```

Comment:
This is certainly a breaking change. Is there some backwards compatibility we can do here, or at least keep the older ones and mark them as deprecated for a deprecation period?

Reply:
sorry, taking a look now. Thanks for making the changes!

Reply:
Okay, if there are some more updates happening, we'd just ask that we do this in a sensible way that takes into account breaking changes and any deprecation periods.

From the sounds of it, I think this PR should be at least a `minor` change in terms of semver.